src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java

```diff
@@ -161,8 +86,8 @@ public void should_read_from_blob_storage_and_save_metadata_in_database_when_zip
         assertThat(processEvents)
             .extracting("container", "zipFileName", "event")
             .contains(
-                tuple(testContainer.getName(), "1_24-06-2018-00-00-00.zip", DOC_UPLOADED),
-                tuple(testContainer.getName(), "1_24-06-2018-00-00-00.zip", DOC_PROCESSED)
+                tuple(testContainer.getName(), ZIP_FILE_NAME_SUCCESS, DOC_UPLOADED),
```

Comment:
Need not have success in file name as it is just file name ZIP_FILE_NAME_SUCCESS

Reply:
will rename on future scenarios in case of using same zip file over and over again - repetitive string is not desired